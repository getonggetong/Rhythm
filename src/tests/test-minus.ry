init() {
	print(-1 - 1);
	print(1 - 2 - 3);
	print(43 - 7);
	print(43 - -7);
	print(A4 - 1);
	print(A4 - 1 - 2);
	print([C4,F4] - 1);
	print([A4,B4] - 1 - 2);
	print([A4.1,B4.2,[C4.4,D4.8]] - 1);
}