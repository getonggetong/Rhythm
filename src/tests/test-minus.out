-2
-4
36
50
G#3.4
F#3.4
[B4.4,E4.4]
[F#3.4,G#3.4]
[G#3.1,A#4.2,[B4.4,C#4.8]]